ruby-monday-blog/app/views/posts/_preview.html.erb

```diff
@@ -0,0 +1,22 @@
+<% hide_tags ||= false %>
```

Comment:
I think I found it. What do you think about using two separate partials? The stuff inside `hide_tags` could be like a 'tags' partial. And the post a 'post/content' partial.